lib/ruby_lsp/ruby_lsp_rails/completion.rb

```diff
@@ -0,0 +1,88 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyLsp
+  module Rails
+    class Completion
+      extend T::Sig
+      include Requests::Support::Common
+
+      sig do
+        override.params(
+          client: RunnerClient,
+          response_builder: ResponseBuilders::CollectionResponseBuilder[Interface::CompletionItem],
+          node_context: NodeContext,
+          dispatcher: Prism::Dispatcher,
+          uri: URI::Generic,
+        ).void
+      end
+      def initialize(client, response_builder, node_context, dispatcher, uri)
+        @response_builder = response_builder
+        @client = client
+        @node_context = node_context
+        dispatcher.register(
+          self,
+          :on_call_node_enter,
+        )
+      end
+
+      sig { params(node: Prism::CallNode).void }
+      def on_call_node_enter(node)
+        call_node = T.must(@node_context.call_node)
+        receiver = call_node.receiver
+        if call_node.name == :where && receiver.is_a?(Prism::ConstantReadNode)
+          handle_active_record_where_completions(node: node, receiver: receiver)
+        end
+      end
+
+      private
+
+      sig { params(node: Prism::CallNode, receiver: Prism::ConstantReadNode).void }
+      def handle_active_record_where_completions(node:, receiver:)
+        resolved_class = @client.model(receiver.name.to_s)
+        return if resolved_class.nil?
+
+        arguments = T.must(@node_context.call_node).arguments&.arguments
+        indexed_call_node_args = T.let({}, T::Hash[String, Prism::Node])
+
+        if arguments
+          indexed_call_node_args = index_call_node_args(arguments: arguments)
+          return if indexed_call_node_args.values.any? { |v| v == node }
+        end
```

Comment:
This early return is meant to prevent suggestions when the user is typing in an argument value within a` .where` in general. e.g when the user is typing `User.where(id: creat` they shouldn't receive a column name suggestion for `created_at`.